### PR TITLE
T34238 kernelci Docker image

### PR DESCRIPTION
**Pull request by gctucker**
Add a `kernelci/kernelci` Docker image with the `kernelci` Python package installed with its dependencies as well as the config files copied to `/etc/kernelci`.

### CODE REVIEW

File: config/docker/kernelci/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2021, 2022 Collabora Limited
```
Comment (nuclearcat) [inline config/docker/kernelci/Dockerfile:3]: Is it 2021?

**Review by nuclearcat — APPROVED** [review]
Builds fine locally for me

File: config/docker/kernelci/Dockerfile (hunk 1 above)
Comment (gctucker) [reply config/docker/kernelci/Dockerfile:3]: I should have mentioned that this Dockerfile was based on the one used by kernelci-pipeline which was written in 2021.  I'll add that in the commit message.

**Comment by gctucker** [comment]
> Builds fine locally for me

This has also been built and used with the new pipeline on staging.

**Pull request merged**